chainservice/builder.go

```diff
@@ -580,9 +594,6 @@ func (builder *Builder) buildBlockSyncer() error {
 				return err
 			}
 			retries := 1
-			if !builder.cfg.Genesis.IsHawaii(blk.Height()) {
-				retries = 4
-			}
```

Comment:
before Hawaii height, delta state may mismatch due to map order, so need to keep this?